src/plugins/layout.ts

```diff
@@ -220,8 +170,7 @@ export default class Layout implements IPlugin {
       this.checkConsistency(actionInfo);
 
       if (this.allowMutation()) {
-        let events = this.processMutations(summary, time);
-        this.processMultipleNodeEvents(events);
+        this.processMutations(summary, time);
       } else {
         debug(`>>> ShadowDom doesn't match PageDOM after mutation batch #${this.mutationSequence}!`);
```

Comment:
>debug [](start = 8, length = 5)

Didn't we use to send an instrumentation event in this case? ShadowDoimINcionsitent...

Reply:
We do, but it's inside the 'allowMutation() -> checkConsistency()' function below. I should probably move this debug statement there as well. 

---
In reply to: [187409318](https://github.com/Microsoft/clarity-js/pull/92#discussion_r187409318) [](ancestors = 187409318)